### PR TITLE
pipewire: add support for aptx and aac

### PR DESCRIPTION
**Pull request by paper42**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

@st3r4g

### CODE REVIEW

**Comment by paper42** [comment]
Yes, those are bluetooth codecs. LDAC is specific to Sony devices afaik, so I can not test it and I don't care about it much right now.

**Review by st3r4g — APPROVED** [review]

File: srcpkgs/pipewire/template [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 pkgname=pipewire
 version=0.3.20
-revision=1
+revision=2
 build_style=meson
 configure_args="-Dman=true -Dgstreamer=true -Ddocs=true -Dsystemd=false
  -Dbluez5=true -Dffmpeg=true -Dpipewire-alsa=true -Dpipewire-jack=true
  -Dudevrulesdir=/usr/lib/udev/rules.d"
-hostmakedepends="doxygen graphviz pkg-config xmltoman"
+hostmakedepends="doxygen graphviz pkg-config xmltoman gettext"
```
Comment (ericonr) [inline srcpkgs/pipewire/template:9]: Could you mention in the commit message that this also adds translations?

File: srcpkgs/pipewire/template [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 makedepends="SDL2-devel ffmpeg-devel gst-plugins-base1-devel jack-devel
- sbc-devel v4l-utils-devel libva-devel libbluetooth-devel ncurses-devel"
+ sbc-devel v4l-utils-devel libva-devel libbluetooth-devel ncurses-devel
+ libopenaptx-devel fdk-aac-devel"
```
Comment (ericonr) [inline srcpkgs/pipewire/template:12]: Might make sense to leave a comment about the possibility of adding LDAC support when packaged?

File: srcpkgs/libopenaptx/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+distfiles="https://github.com/pali/libopenaptx/releases/download/${version}/libopenaptx-${version}.tar.gz"
+checksum=51a13a34bb188a87ab1deb64a090ab42a4755d7b02c2bdaf3b8136f749b71b84
+
+libopenaptx-devel_package() {
+	depends="libopenaptx>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+		vmove usr/lib/pkgconfig
+		vmove usr/lib/*.so
```
Comment (ericonr) [inline srcpkgs/libopenaptx/template:19]:
```suggestion
		vmove "usr/lib/*.so"
```

**Pull request merged**